### DISCUSSION TITLE
Configuration option and/or ENV var for base path

### DISCUSSION THREAD

**Discussion by Industrial** [directus/directus#5422] [Feature Requests]
Hi!

I would like to have a configuration option called `basePath: string` in https://github.com/directus/directus/blob/main/api/src/app.ts#L49 to set the **base path** of the installation. This would help a lot with configuring Directus behind a proxy. See https://github.com/directus/directus/discussions/5309#discussioncomment-686668

If not possible with a configuration option, then maybe with an ENV var called `BASE_PATH`.

I could take a look at this tonight and see if I can make a PR for it :)

Good idea?

**Comment by rijkvanzanten** [comment]
The base path of the API endpoints? Normally that's handled by a proxy server in front 🤔 

We already have `PUBLIC_URL` as well, which is more or less setting base path on the app side of things

**Comment by azrikahar** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
